.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
-*       @alexanderbez @cmwaters @marbar3778 @tychoish @williambanfield
```

Comment:
Actually, most of the users (everyone but bez) here had an incorrect username, or not access to this repo.